user_guide/intro_nav_search.rst

```diff
@@ -0,0 +1,90 @@
+.. _user-guide-getting-started-search:
+
+Search
+======
+
+Search is the fastest way to find a specific record you need. To find a record in OroCRM:
+
+# Click the search icon in the top of the screen – |IcSearch|
+# Type in the search key the text field
+# Click Go or hit Enter on the keyboard
+
+
+For example, we are searching for "Ann" all over the system:
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_1.png
+
+|
+
+
+These are our search results:
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_2.png
+
+|
+
+The search returned 28 records, 2 of which are Accounts, 2 – Contacts, 6 are Magento Shopping Carts, etc. You can scroll 
+through the entire list of results or select a specific entity in order to find the desired record faster.
```

Comment:
Edit: The search returned 28 records, two of which are Accounts, two are Contacts, six are Magento Shopping Carts, etc. You can scroll through the entire list of results or select a specific entity in order to find the desired record faster.